tor/core/admin_commands.py

```diff
@@ -109,36 +184,18 @@ def process_blacklist(reply, config):
             )
         )
 
+        return results
 
-def reload_config(reply, config):
-    if not from_moderator(reply, config):
-        logging.info(
-            '{} just issued a reload command. No.'.format(reply.author.name)
-        )
 
-        reply.reply(_(random.choice(config.no_gifs)))
-    else:
-        logging.info(
-            'Reloading configs at the request of {}'.format(reply.author.name)
-        )
-        reply.reply(
-            'Config reloaded!'
-        )
-        initialize(config)
-        logging.info('Reload complete.')
+def reload_config(reply, config):
+    logging.info(
+        'Reloading configs at the request of {}'.format(reply.author.name)
+    )
+    initialize(config)
+    logging.info('Reload complete.')
 
+    return 'Config reloaded!'
 
-def update_and_restart(reply, config):
-    if not from_moderator(reply, config):
 
-        reply.reply(_(random.choice(config.no_gifs)))
-        logging.info(
-            '{} just issued update. No.'.format(reply.author.name)
-        )
-    else:
-        pass
-        # TODO: This does not currently function on our primary box.
-        # # update from repo
-        # sh.git.pull("origin", "master")
-        # # restart own process
-        # os.execl(sys.executable, sys.executable, *sys.argv)
+def ping(reply, config):
+    return "Pongity ping pong pong!"
```

Comment:
this response does not match the docstring :P

Reply:
Done!